src/engine/renderer/tr_shader.cpp

```diff
@@ -4352,7 +4352,8 @@ static void CollapseStages()
 					ASSERT_GE(lightStage, 0);
 					Log::Debug("found glow map with custom lightmap stage in '%s' shader, not collapsing", shader.name);
 					stages[ glowStage ].type = stageType_t::ST_COLORMAP;
-					stages[ glowStage ].bundle[ TB_COLORMAP ] = stages[ glowStage ].bundle[ 0 ];
+					// TB_COLORMAP == 0
+					// stages[ glowStage ].bundle[ TB_COLORMAP ] = stages[ glowStage ].bundle[ 0 ];
```

Comment:
I'd say drop this commit. The compiler can figure out on its own that the assignment is not needed and optimize it out.

Reply:
ok, I still think a comment may be useful in anyway…